src/gui/DatabaseWidget.cpp

```diff
@@ -755,7 +755,7 @@ void DatabaseWidget::entryActivationSignalReceived(Entry* entry, EntryModel::Mod
 void DatabaseWidget::switchToEntryEdit()
 {
     Entry* entry = m_entryView->currentEntry();
-    Q_ASSERT(entry);
+    
```

Comment:
Some code clearly calls switchToGroup/EntryEdit() when it does not make sense. I'd rather get to the bottom of it and fix the root cause.

The if (...) return is only meant as a safety measure to not crash the program on non-fatal bugs.